docs/user-flows.md

```diff
@@ -1,262 +1,157 @@
 
-#### Benefits
 
-- Provides an overall performance score and all metrics.
-- Contains the most advice of all report types (both time-based and state-based audits are available).
 
-#### Limitations
 
-- Cannot analyze form submissions or single page app transitions.
-- Cannot analyze content that isn't available immediately on page load.
 
-#### Use Cases
+### Navigation mode
 
-- Obtain a Lighthouse Performance score.
-- Measure Performance metrics (First Contentful Paint, Largest Contentful Paint, Speed Index, Time to Interactive, Cumulative Layout Shift, Total Blocking Time).
-- Assess Progressive Web App capabilities.
+In DevTools, navigation is easy: ensure it's the selected mode and then click _Analyze page load_.
 
-#### Triggering a navigation via user interactions
+![navdt](https://user-images.githubusercontent.com/39191/168929174-11311144-ce9b-4124-9a52-0423a073b9fe.png)
 
-Instead of providing a URL to navigate to, you can provide a callback function. This is useful when you want to audit a navigation where the destination is unknown before navigating.
+> Note: DevTools only generates a report for a standalone navigation. To combine it with other steps for a multi-step user flow report, [use the Node API](#creating-a-flow).
 
-> Aside: Lighthouse typically clears out any active Service Worker and Cache Storage for the origin under test. However, in this case, as it doesn't know the URL being analyzed, Lighthouse cannot clear this storage. This generally reflects the real user experience, but if you still wish to clear the Service Workers and Cache Storage you must do it manually.
-
-This callback function _must_ perform an action that will trigger a navigation. Any interactions completed before the callback promise resolves will be captured by the navigation.
-
-#### How to Use
-
-<details>
-<summary>
-DevTools
-</summary>
-
-1. Go to the page you want to test
-2. Select "Navigation (Default)" as your mode
-3. Click "Analyze page load"
-
-> Note: DevTools only generates a report for a standalone navigation, it cannot be combined with other steps in a user flow report.
-
-![Lighthouse DevTools panel in navigation mode](https://user-images.githubusercontent.com/6752989/168673207-1e901e72-3461-4bae-a581-e80963beea54.png)
-</details>
-
-<details>
-<summary>Node API</summary>
+#### Navigations in the Node.js API
 
 ```js
 import {writeFileSync} from 'fs';
 import puppeteer from 'puppeteer';
-import api from 'lighthouse/lighthouse-core/fraggle-rock/api.js';
+import lighthouse from 'lighthouse/lighthouse-core/fraggle-rock/api.js';
```

Comment:
yeah that makes sense. but i also think `api` is too general. it looks 'fine' on this line but then later with `api.startFlow` it's lost its identity

`lighthouseapi` ? `lhapi`?  

we should also fix the export so that `import lighthouse from 'lighthouse'` actually would work for all this. i feel good about doing that for v10

Reply:
> we should also fix the export so that import lighthouse from 'lighthouse' actually would work for all this. i feel good about doing that for v10

Ok, in that case we can leave this with plans to update the docs once `startFlow` is exported via the default entry point.